### DISCUSSION TITLE
Can "severity" be defined based on evaluation?

### DISCUSSION THREAD

**Discussion by ksanjeet** [trento-project/wanda#274] [Q&A] [answered]
In one of the check it is required that based on the result of evaluation, severity should be changed. Is it possible to change the value of severity after the evaluations.

**Comment by nelsonkopliku** [comment, marked as answer]
No. At the moment severity is static.

**Reply by ksanjeet** [reply]
Thank you for the quick answer.

**Reply by ksanjeet** [reply]
I am closing the discussion for now, but I want to mention that it would make check writing easier if severity can be changed based on evaluation.
The example for which I raised this question was about the value of the score of an order constraint in ASCS-ERS cluster, which can be "Optional" (desired) and then other possible value is "2000" (which is currently allowed but in future it will be made obsolete), so the idea was , when "Optional" is found then "Passing", when "2000" is found then "Warning" and if the constraint is not found then "Critical".
